sdk/keyvault/keyvault-secrets/test/public/lro.recover.spec.ts

```diff
@@ -94,7 +94,10 @@ describe("Secrets client - Long Running Operations - recoverDelete", () => {
 
   // On playback mode, the tests happen too fast for the timeout to work
   it("can attempt to recover a deleted secret with requestOptions timeout", async function (this: Context) {
-    recorder.skip(undefined, "Timeout tests don't work on playback mode.");
+    if (!isLiveMode()) {
```

Comment:
Timeout tests _should_ be working in the new recorder, but are still flaky. We should investigate. I will log an issue.

Reply:
#23509